pages/team.js

```diff
@@ -1,70 +1,125 @@
 import React, { Component } from 'react'
 import { connect } from 'unistore/react'
-import { actions } from '../lib/store'
-import CampaignCard from '../components/campaigns/CampaignCard'
-import { sortBy, prop } from 'ramda'
-import Table from '../components/common/Table'
+import { pathOr, head, flatten, fromPairs } from 'ramda'
+import { compareAsc, distanceInWordsToNow, format as dateFormat, getYear } from 'date-fns'
+import Slugger from 'github-slugger'
 
-const tableSchema = {
-  'headers': {
-    'name': { type: 'namelink', accessor: 'full_name' },
-    'user-id': { type: 'string', accessor: 'osm_id' }
-  },
-  columnOrder: [
-    'name',
-    'user-id'
-  ],
-  displaysTooltip: [
-    'user-id'
-  ]
-}
+import ScoreboardPanel from '../components/ScoreboardPanel'
+import TeamStatsTable from '../components/campaign/CampaignTable'
+import Table from '../components/common/Table'
+import Blurb from '../components/teams/TeamBlurb'
+import { formatDecimal, normalizeHashtag } from '../lib/utils/format'
+import { actions } from '../lib/store'
 
 export class Team extends Component {
   componentDidMount () {
     this.props.getTeam(this.props.id)
+    this.slugger = new Slugger()
   }
 
   render () {
     const { team } = this.props
     if (!team) return <div />
 
-    let idMap = {}
-    if (team.users.length) {
-      idMap = Object.assign(...team.users.map(({ osm_id, full_name }) => ({ [full_name]: osm_id })))
-    }
-
+    const {
+      name: teamName,
+      hashtag,
+      created_at: teamCreated,
+      lastRefreshed,
+      osmesaStats,
+      campaigns
+    } = team
+    const { buildingsMappedCount, poiCountMappedCount, roadsKmMapped,
+      waterWaysKmMapped, coastlinesKmMapped } = calculateBlurbStats(osmesaStats.teamStats)
+    const allMemberEditTimes = flatten(osmesaStats.memberStats.map(({ edit_times }) => edit_times))
+    const firstYearEdited = getYear(head(allMemberEditTimes.map(t => t.day).sort(compareAsc)))
+    const exportDataFilename = this.slugger.slug(teamName || 'team export', false)
```

Comment:
What's the rationale behind using `github-slugger`?

Reply:
@kamicut just personal preference for having filenames that are not full of spaces and special characters. Would be happy to remove it if you do not like it.

Reply:
It was just out of curiosity